sdk/react/index.ios.js

```diff
@@ -61,7 +61,7 @@ class OoyalaSkin extends React.Component {
   };
 
   componentWillMount() {
-    console.log("componentWillMount");
+    Log.verbose("componentWillMount");
```

Comment:
I think we can remove this log, not sure how useful it is for us. What do you think?